Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Card.css';
+
+const CardContainer = () => {
+    return (
+      <div className = "cardContainer">
+        <a href = "reactjs.org">
+            <img src = "https://ibin.co/3wnC6SgIOJud.png" alt = "" className = "cardContainerImg"/>
+            <p>Get started with React</p>
```

Comment:
This "Get Started" header and subsequent p tags should be abstracted out to the HeaderContent component.